provider/resource_server.go

```diff
@@ -40,6 +43,49 @@ func resourceServer() *schema.Resource {
 				Elem:        resourceInterface(),
 				Description: "Private interface properties",
 			},
+			"os_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				Description: "OS id, use the following command to select one " +
+					"`curl -s -H \"Authorization: Bearer $ONLINE_TOKEN\"  " +
+					"https://api.online.net/api/v1/server/operatingSystems/{server id}`",
+			},
+			"user_login": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "User login",
+			},
+			"user_password": &schema.Schema{
```

Comment:
This should be marked as a sensitive value, we don't want passwords to be exposed in the plan logs